docs/REST API/rest-api.md

```diff
@@ -4276,6 +4276,8 @@ OS vulnerability data is currently available for Windows and macOS. For other pl
 
 ### Get host's software
 
+> The new keys/values added in the app management features are experimental and may change. You can find the upcoming breaking changes [here](https://github.com/fleetdm/fleet/pull/19291/files#diff-7246bc304b15c8865ed8eaa205e9c244d0a0314e4bae60cf553dc06147c38b64L4304-L4311).
```

Comment:
```suggestion
> **This feature is still experimental.** 
>
>The new keys/values added in the app management features are experimental and may change. You can find the upcoming breaking changes [here](https://github.com/fleetdm/fleet/pull/19291/files#diff-7246bc304b15c8865ed8eaa205e9c244d0a0314e4bae60cf553dc06147c38b64L4304-L4311).
```

What do you think of making this callout clearer like Mike's example from Sails?